elasticsearch/utils/es_seed_acl

```diff
@@ -6,6 +6,13 @@ ES_CLUSTER_HOST=${ES_CLUSTER_HOST:-"localhost"}
 ES_CLUSTER_PORT=${ES_CLUSTER_PORT:-9300}
 MAX_WAIT_SEED=${MAX_WAIT_SEED:-$((60*60*24*7))} #one week
 
+NODES=$(es_util --query=_cat/nodes | wc -l)
```

Comment:
will this work how we anticipate? isn't there a chance we haven't yet joined the cluster at this point in time during the node's initial initialization?

Reply:
my quick test for an "error" condition would produce a value of zero meaning we would have zero replicas.  Given we run this N times for each node it should settle on a value of 1 for 3 or more nodes.  Ideally the operator will control this properly when it inits the cluster